releases.md

```diff
@@ -42,10 +42,10 @@ Further documentation available:
 
 ### v0.23
 
-- **Latest Release**: [v0.23.0][v0-23-0] (2023-03-02) ([docs][v0-23-0-docs], [examples][v0-23-0-examples])
+- **Latest Release**: [v0.23.1][v0-23-1] (2023-04-03) ([docs][v0-23-1-docs], [examples][v0-23-1-examples])
```

Comment:
I believe now latest release is v0.24.0 

Can you add that info as well

Reply:
Done.